docs/en/sdk-manual/web.md

```diff
@@ -70,167 +236,220 @@ Clickstream Web SDK supports the following data types:
 
-In Clickstream Web SDK, we do not limit the total time of a session, as long as the time between the next entry of the browser and the last exit time is within the allowable timeout period, we consider the current session to be continuous.
+The `_session_start` event triggered when the website open for the first time, or the browser was open to the foreground and the time between the last exit exceeded `session_time_out` period, and the following are session-related attributes.
 
-**_session_start**: When the website open for the first time, or the browser was open to the foreground and the time between the last exit exceeded `session_time_out` period.
+1. _session_id: We calculate the session id by concatenating the last 8 characters of uniqueId and the current millisecond, for example: dc7a7a18-20230905-131926703.
 
-**_session_duration**: We calculate the `_session_duration` by minus the current event create timestamp and the session's `_session_start_timestamp`, this attribute will be added in every event during the session.
+2. _session_duration : We calculate the session duration by minus the current event create timestamp and the session's `_session_start_timestamp`, this attribute will be added in every event during the session.
 
-**session_time_out**: By default is 30 minutes, which can be customized through the configuration api.
+3. _session_number : The auto increment number of session in current browser, the initial value is 1
 
-**_session_number**: The total number of session distinct by session id, and `_session_number` will be appeared in every event's attribute object.
+4. Session timeout duration: By default is 30 minutes, which can be customized through the [configuration](#other-configurations) api.
 
-#### User engagement definition
+### Page view definition
 
-In Clickstream Web SDK, we define the `user_engagement` as the webpage is in focus at least one second.
+In Clickstream Web SDK, we define the `_page_view` as an event that record the user's browsing path of page, when a page transition started, the `_page_view` event will be recorded when meet any of the following conditions:
 
-**when to send**: We send the event when the browser is close, hide or navigate to another  web page in different domain.
+1. No page was previously set.
+2. The new page title differs from the previous page title.
+3. The new page url differ from the previous page url.
 
-**engagement_time_msec**: We count the time from when the web page is visible to when the web page is hidden.
+This event listens for `pushState`, `popState` in history, and `replaceState` of window to judgment the page transition. In order to count page browsing path, we use `_page_referrer`(last page url) and `page_referrer_title` to link the previous page. In addition, there are some other attributes in page view event.
```

Comment:
In order to count browsing path --> in order to track

Reply:
fixed.